docs/kafka/nodejs-kafka/README.adoc

```diff
@@ -183,6 +183,8 @@ The values are described as follows:
 endif::[]
 +
 In this case, observe that the Node.js application uses the SASL/PLAIN authentication method (that is, the value of `KAFKA_SASL_MECHANISM` is set to `plain`). This means that the application uses only the client ID and client secret to authenticate with the Kafka instance. The application doesn't require an authentication token.
++
```

Comment:
Could we get rid of the instruction to use PLAIN altogether here? So then we could add something like this: In the `rhoas.env` file, set the SASL connection mechanism and the Kafka instance client credentials as shown in the following configuration. Replace the values with your own credential information. {product-kafka} supports the SASL/OAUTHBEARER mechanism for authentication, which is the recommended authentication mechanism to use.

KAFKA_HOST=<bootstrap_server>
RHOAS_SERVICE_ACCOUNT_CLIENT_ID=<client_id>
RHOAS_SERVICE_ACCOUNT_CLIENT_SECRET=<client_secret>
KAFKA_SASL_MECHANISM=oauthbearer
RHOAS_TOKEN_ENDPOINT_URL=https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token

Reply:
For the tutorial I'm happy to only mention OAUTHBEARER, even though the code will support both.

Reply:
@AedinC Is it ok to push it in a commit ? and have a review on that too? Practically i'll copy paste the above text you wrote and replace the `plain` instructions.
```
Could we get rid of the instruction to use PLAIN altogether here? So then we could add something like this: In the rhoas.env file, set the SASL connection mechanism and the Kafka instance client credentials as shown in the following configuration. Replace the values with your own credential information. {product-kafka} supports the SASL/OAUTHBEARER mechanism for authentication, which is the recommended authentication mechanism to use.

KAFKA_HOST=<bootstrap_server>
RHOAS_SERVICE_ACCOUNT_CLIENT_ID=<client_id>
RHOAS_SERVICE_ACCOUNT_CLIENT_SECRET=<client_secret>
KAFKA_SASL_MECHANISM=oauthbearer
RHOAS_TOKEN_ENDPOINT_URL=https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token
```

Reply:
@pacostas Aedin is on PTO this week, so I'll do another review of the updates for you :+1: . I actually worked with @mhdawson on the original version of this quick start.

Reply:
Its a very interesting tutorial and a hot (not kafka) topic these days :p  
@jbyrne-redhat Here is the commit I just pushed, with the text pasted from Aedin https://github.com/redhat-developer/app-services-guides/pull/600/commits/73c2264fc704206b5568ffcc016a4d014f77b9ba